exercises/I-array-properties/exercise.js

```diff
@@ -6,7 +6,7 @@
 */
 
 function isEmpty(arr) {
-  return; // complete this statement
+  return arr.length === 0; // complete this statement
```

Comment:
Good use of returning a statement to return a Boolean value; this is a better pattern to use than something like 
```
If (arr.length === 0) {
   return true
} else {
   return false
}
```

Good job